src/init/autoDetectProject.ts

```diff
@@ -66,7 +61,9 @@ export function iconsDisabled(name: string): boolean {
 export function isProject(projectJson: any, name: string): boolean {
   switch (name) {
     case 'ng':
-      return (projectJson.dependencies && (projectJson.dependencies['@angular/core'] != null)) || false;
+      return (projectJson.dependencies && (projectJson.dependencies['@angular/core'] != null)) ||
+        (projectJson.devDependencies && (projectJson.devDependencies['@angular/core'] != null)) ||
+        false;
```

Comment:
Revising my comment. It turns out that the `guard clause` is indeed needed (praise unit testing).
You see the case is that:
`projectJson.dependencies && (projectJson.dependencies['@angular/core'] != null)` can be `false` and `projectJson.devDependencies && (projectJson.devDependencies['@angular/core'] != null)` can be `undefined`.
`false || undefined` returns `undefined`, thus `|| false` is needed.